src/Stratis.Bitcoin.Tests/P2P/PeerSelectorTests.cs

```diff
@@ -358,41 +377,41 @@ public void PeerSelector_ReturnPeersForGetAddrPayload_Scenario3()
 
             for (int i = 1; i <= 15; i++)
             {
-                var ipAddress = IPAddress.Parse(string.Format("::ffff:192.168.0.{0}", i));
+                IPAddress ipAddress = IPAddress.Parse(string.Format("::ffff:192.168.0.{0}", i));
                 peersToAdd.Add(new IPEndPoint(ipAddress, 80));
             }
 
-            var peerFolder = AssureEmptyDirAsDataFolder(Path.Combine(AppContext.BaseDirectory, "PeerAddressManager"));
-
-            var peerAddressManager = new PeerAddressManager(DateTimeProvider.Default, peerFolder, this.extendedLoggerFactory);
+            PeerAddressManager peerAddressManager = this.CreatePeerAddressManager();
             peerAddressManager.AddPeers(peersToAdd.ToArray(), IPAddress.Loopback);
 
             for (int i = 1; i <= 7; i++)
             {
-                var ipAddress = IPAddress.Parse(string.Format("::ffff:192.168.0.{0}", i));
+                IPAddress ipAddress = IPAddress.Parse(string.Format("::ffff:192.168.0.{0}", i));
                 peerAddressManager.PeerAttempted(new IPEndPoint(ipAddress, 80), DateTime.UtcNow.AddHours(-2));
             }
 
-            var peers = peerAddressManager.PeerSelector.SelectPeersForGetAddrPayload(15);
+            IEnumerable<PeerAddress> peers = peerAddressManager.PeerSelector.SelectPeersForGetAddrPayload(15);
             Assert.Equal(15, peers.Count());
             Assert.Equal(7, peers.Count(p => p.Attempted));
             Assert.Equal(8, peers.Count(p => p.Fresh));
         }
 
         /// <summary>
+        /// <para>
```

Comment:
In the future so you know comments on unit-tests are optional

Reply:
Yep cool - I was just following the style of the tests already there.